src/dfx/src/lib/canister_info/assets.rs

```diff
@@ -11,7 +11,7 @@ pub struct AssetsCanisterInfo {
 
     output_wasm_path: PathBuf,
     output_idl_path: PathBuf,
-    output_assets_path: PathBuf,
+    output_frontend_path: PathBuf,
```

Comment:
All changes in this file should be reverted
```suggestion
    output_assets_path: PathBuf,
```